Improved_Switch_Table_Creation/readme.md

```diff
@@ -21,3 +21,6 @@ $ riscv64-unknown-elf-gcc -o output/main.o @bld_args main.c
 $ riscv64-unknown-elf-gcc -o output/main.elf @lnk_args output/main.o
 $ riscv64-unknown-elf-objdump -D output/main.elf > output/main_dump 2>&1
 ```
+
```

Comment:
i would have removed the --sysroot and --gcc-toolchain from the file and ask the use to put it in command line but it is up tp you:
### Build with LLVM
riscv64-unknown-elf-gcc -o output/main.elf @lnk_args output/main.o --sysroot=PATH1 --gcc-toolchain=PATH2